ldes-server-integration-test/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/LdesServerSteps.java

```diff
@@ -362,4 +363,10 @@ public void theResponseFromRequestingTheUrlDoesContainAJsonFile(String message,
 				.andReturn().getResponse();
 		assertTrue(response.getContentAsString().contains("%s{collection=\"%s\"} %s".formatted(message, collection, value)));
 	}
+
+	@And("the background processes did not fail")
+	public void theBackgroundProcessesDidNotFail() {
+		final List<String> failedStepExecutions = jdbcTemplate.queryForList("SELECT step_name FROM batch_step_execution WHERE status = 'FAILED'", String.class);
+		assertThat(failedStepExecutions).isEmpty();
+	}
```

Comment:
but .. this shouldn't matter if you get the correct results..? 
Sure we expect clean steps with no errors, but it doesn't really sound like a step i expect in a feature file of compaction